frontend/src/index.css

```
@@ -102,120 +511,221 @@
+  }
+
+  .fade-in-up {
+    animation: fadeInUp 340ms cubic-bezier(0.22, 1, 0.36, 1);
+  }
 }
 
-@keyframes da-pulse {
-  0% { transform: scale(2.2); opacity: 0.5; box-shadow: 0 0 8px currentcolor; }
-  40% { transform: scale(1.4); opacity: 0.8; }
-  100% { transform: scale(1); opacity: 1; box-shadow: none; }
+@keyframes highlightFade {
```

Stylelint (17.7.0)

[error] Expected keyframe name "highlightFade" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)